src/ngx_http_lua_worker_thread.c

```diff
@@ -0,0 +1,356 @@
+/*
+ * Copyright (C) Jinhua Luo (kingluo)

```

Comment:
You need to grant the copyright to the owners of this project for us to merge your code.